orbs/pulumi.yml

```diff
@@ -50,6 +50,42 @@ commands:
             export PULUMI_ACCESS_TOKEN="<< parameters.access-token >>"
             pulumi login --cloud-url << parameters.cloud-url >>
 
+  # stack init command
+  stack_init:
+    parameters:
+      stack:
+        type: string
+        description: "Name of the stack to initialize."
+      secrets_provider:
+        type: string
+        description: "The type of the provider that should be used to encrypt and decrypt secrets (possible choices: default, passphrase, awskms, azurekeyvault, gcpkms, hashivault) (default "default")."
+        default: "default"
+      working_directory:
+        type: string
+        description:
+            "The relative working directory to use, i.e. where your Pulumi.yaml is located."
+        default: "."
+    steps:
+      - run:
+          name: "pulumi stack init --stack << parameters.stack >> --secrets-provider << parameters.secrets_provider >>"
+          command: pulumi stack init --stack << parameters.stack >> --secrets-provider << parameters.secrets_provider >> --cwd << parameters.working_directory >>
+
+  # stack rm command
+  stack_rm:
+    parameters:
+      stack:
+        type: string
+        description: "Name of the stack to remove."
+      working_directory:
+        type: string
+        description:
+            "The relative working directory to use, i.e. where your Pulumi.yaml is located."
+        default: "."
+    steps:
+      - run:
+          name: "pulumi stack rm << parameters.stack >>"
+          command: pulumi stack rm << parameters.stack >> --force --yes --cwd << parameters.working_directory >>
```

Comment:
We should remove `--yes` entirely. When `pulumi` is running in a non-interactive session (e.g. a CI/CD job), then it will automatically skip prompts. (So the `--yes` command doesn't hurt anything, it just isn't needed. So we've omitted them to be more concise.)